zulipterminal/model.py

```diff
@@ -825,6 +825,20 @@ def fetch_message_history(
         display_error_if_present(response, self.controller)
         return list()
 
+    def fetch_read_receipts(self, message_id: int) -> Optional[List[int]]:
+        """
+        Fetches read receipts of a message using its ID.
+        """
```

Comment:
I understand the endpoint is termed read receipts, and that is not a bad name for this function, but internally the result we obtain is a list of user ids. It would be more readable if this method was phrased to indicate that. This avoids the question 'what is a read receipt'?

This and a test for this would be good in a separate first/early commit.

Reply:
Yep that's a good point. Changing it to `Fetches user IDs of users who've read a message using its ID.`

Re the commit structure, I was going to add the model function and test in one commit and the UI part in a separate commit but couldn't figure out the test case error so doing this for now. Will change it to the proper commit structure as soon as I figure out the test case

Reply:
Just to confirm, I was also meaning ideally the name of the function, not just the docstring.

Reply:
Would you have any suggestions for the name? Something like `fetch_message_reader_list` or `fetch_message_read_list`?

Reply:
This is fine for now, though you seem to have changed the return type?

Reply:
I would use a name like `fetch_messaged_read_receipt_user_ids` for this sort of thing -- but I'm particularly a fan of names that make the expected types clear, and not particularly focused on avoiding overly long names :)